Formula/qt.rb

```diff
@@ -40,6 +41,8 @@ def install
       -pkg-config
       -dbus-runtime
       -proprietary-codecs
+      -I#{Formula["molten-vk"].opt_include}
```

Comment:
All other packages are autodetected, why are the molten-vk headers not autodetected?

Reply:
I'm not sure. I followed this blog post https://www.qt.io/blog/2018/05/30/vulkan-for-qt-on-macos 
Maybe only the path to molten-vk’s vulkan headers in `libexec/include` is required. I’m esting this locally now.